cmd/launcher/gui/gui_progress_control_test.go

```diff
@@ -0,0 +1,30 @@
+package gui
+
+import (
+	"testing"
+	"time"
+)
+
+func TestRateString(t *testing.T){
+	//pf := rateString{ "Private Field" }
```

Comment:
please remove when not needed :)

Reply:
Funny enough, changed it already but forgot to add it to the commit!